app/models/exercise.rb

```diff
@@ -27,6 +27,8 @@ class Exercise < ActiveRecord::Base
     where(course_id: course.id, gdocs_sheet: gdocs_sheet)
   }
 
+  enum disabled_status: [:enabled, :disabled]
```

Comment:
I'd just make this a boolean, to be consistent with the other code if nothing else.